tests/integration/vlan_test.py

```diff
@@ -90,9 +91,39 @@ def test_add_and_remove_two_vlans_on_same_iface(eth1_up):
     assert not current_state[Interface.KEY]
 
 
+def test_two_vlans_on_eth1_change_mtu(eth1_up):
+    desired_state = statelib.show_only(('eth1',))
+    vlans_state = copy.deepcopy(TWO_VLANS_STATE)
```

Comment:
Instead of using `deepcopy` it will be nicer to use a creator helper: `create_two_vlans_state` that replaces `TWO_VLANS_STATE`.

Reply:
Done. Thanks!